params/safety_config.yaml

```diff
@@ -0,0 +1,11 @@
+safety_bonds:
```

Comment:
They will find out very quickly if they aren't following the convention. It's also typical to load your included modules before your own code in at least programming languages. I don't see why roslaunch files should be essentially different.